classes/blocks/class-covers.php

```diff
@@ -420,21 +420,25 @@ private static function populate_posts_for_cfc( &$fields ) {
 
 		if ( ! empty( $posts ) ) {
 
-			foreach ( $posts as $post ) {
+			foreach ( $posts as $post_data ) {
```

Comment:
Did a quick workaround to avoid `post_content` and more data from being included in the attributes.

It's technically not needed but it made the attributes very big too work with.